metadata.json

```diff
@@ -14,7 +14,7 @@
     },
     {
       "name": "puppetlabs/stdlib",
-      "version_requirement": ">= 3.4.0 < 6.0.0"
+      "version_requirement": ">= 5.0.0 < 6.0.0"
```

Comment:
You can remove this change as the module has now been bumped to 5.5.0.

Reply:
stdlib version 3.4.0 doesn't have the needed `Stdlib::Filemode` type alias.

Reply:
Nevertheless, I'll remove the change as you request.

Reply:
Oh woops misread that as the puppet version not stdlib :)